installer/win/Symphony-x64.aip

```diff
@@ -126,6 +126,7 @@
     <ROW Component="cld.node" ComponentId="{D9F74D7B-B2E4-4ED8-A023-A54A98AE3F96}" Directory_="Release_Dir" Attributes="256" KeyPath="cld.node" Type="0"/>
     <ROW Component="d3dcompiler_47.dll" ComponentId="{C7B87C02-3116-43A8-A70B-3592B70E6AC8}" Directory_="APPDIR" Attributes="256" KeyPath="d3dcompiler_47.dll"/>
     <ROW Component="ffmpeg.dll" ComponentId="{A1C4A332-3490-44D8-A5C9-9523889B488B}" Directory_="APPDIR" Attributes="256" KeyPath="ffmpeg.dll"/>
+    <ROW Component="freediskspace.exe" ComponentId="{012071B8-758A-4A24-8A13-A48476F9F6FC}" Directory_="library_Dir" Attributes="0" KeyPath="freediskspace.exe"/>

```

Comment:
Is there no other way to do this? If not, what operations are this file doing? Can we not use a DLL instead?

@VishwasShashidhar @lneir Can you suggest an alternative if any?

Reply:
@VikasShashidhar I was using Windows 'Fsutil' before but it had permission issues so thought of using the `DriveInfo.AvailableFreeSpace Property` and created an execution file and now this won't have any permission issue.